todo_app/trello_items.py

```diff
@@ -16,102 +18,39 @@ def Get_trello_APITODOID():
 def Get_trello_APIDOING():
     return os.getenv('DOING')
     
-
-def Get_items_from_trello():
-   
-    #url = 'https://api.trello.com/1/members/me/boards'
-
-    url = f"https://api.trello.com/1/boards/{os.getenv('BOARDID')}/cards"
-
-    query = {
-    'key': Get_trello_APIkey(),
-    'token': Get_trello_APIToken()
-    }
-
-    response = requests.request(
-    "GET",
-    url,
-    params=query
-    )
+def Get_items_from_mongodb():
+  client=pymongo.MongoClient(os.getenv('MONGODB')) 
+  db = client[os.getenv('MONGODBDATABASE')]
+  collection = db.test_collection
+  todoitemList=[]
+  for todoitem in collection.find():
+      todoitemList.append(Item(todoitem['_id'],todoitem['name'],todoitem['status']))
+  return todoitemList    
+
+ 
+def Create_iten_from_mongodb(createitem):
+    client=pymongo.MongoClient(os.getenv('MONGODB')) 
+    db = client[os.getenv('MONGODBDATABASE')]
+    collection = db.test_collection
+    collection.insert_one({'name': createitem,'status': 'TODO'})
 
     
-    response=response.json()
-    for card in response:
-        if card['idList']==Get_trello_APITODOID():
-            card['status']='TODO'
-        elif card['idList']== Get_trello_APIDOING():
-            card['status']='DOING'
-
 
-    idList=[]
-    for card in response:
-        idList.append(Item(card['id'],card['name'],card['status'],card['idList']))
-        
-    return idList
-
-def create_card(name_of_card):
-    url = "https://api.trello.com/1/cards"
-    query = {
-    'key': Get_trello_APIkey(),
-    'token':Get_trello_APIToken(),
-    'idList': Get_trello_APITODOID(),
-    'name': name_of_card
-    }
-    response = requests.request(
-    "POST",
-    url,
-    params=query
-    )
-
-def move_card(id):
-    url = f"https://api.trello.com/1/cards/{id}"
+    
+    
+    
 
-    headers = {
-    "Accept": "application/json"
-    }
+def move_item_from_mongodb(id):
+    client=pymongo.MongoClient(os.getenv('MONGODB')) 
+    db = client[os.getenv('MONGODBDATABASE')]
+    collection = db.test_collection
+    filter = { '_id':  ObjectId(id) }
+    newvalues = { "$set": { 'status': 'DONE' } }
+    collection.update_one(filter, new_values)
```

Comment:
A slight mismatch of names means this isn't currently working:
```suggestion
    new_values = { "$set": { 'status': 'DONE' } }
    collection.update_one(filter, new_values)
```